src/js/components/organisms/hero-product/hero-product.scss

```diff
@@ -0,0 +1,128 @@
+@import "~scss/component";
+
+$imageLinkSizeMobile: $buildingUnit * 7;
+$imageLinkSizeDesktop: $buildingUnit * 10;
+
+.cta__block--hero {
+    align-items: flex-start;
+    @include min-tablet() {
+        align-items: center;
+    }
+}
+
+.hero--product {
+    @extend %reset-last-margin;
+
+    .hero__container {
+        @include max-mobile() {
+            padding-top: $buildingUnit;
+        }
+    }
+
+    .hero__image--basic,
+    .hero__image--product{
```

Comment:
It shouldn't be necessary to wrap a modifier(hero__image--product) in a modifier (hero--product)

Reply:
Fixed